src/main/java/org/opensearch/security/ssl/http/netty/SecuritySSLNettyHttpServerTransport.java

```diff
@@ -94,6 +94,12 @@ public void onException(HttpChannel channel, Exception cause0) {
         super.onException(channel, cause0);
     }
 
+    // TODO: Disable since changes in core aren't avaliable yet
```

Comment:
Is there a PR associated with this change in core?

Reply:
Oh, I should pull that commented out section from here, you can see the core changes [1] on PR 10261

- [1] https://github.com/opensearch-project/OpenSearch/pull/10261